### PR TITLE
{bio}[goolf/1.5.14] samblaster 0.1.21

### PR DESCRIPTION
**Pull request by jordiblasco**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/s/samblaster/samblaster-0.1.21-goolf-1.5.14.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Authors::   Jordi Blasco <jordi.blasco@nesi.org.nz>
+# New Zealand eScience Infrastructure (NeSI)
+
+name = "samblaster"
+version = "0.1.21"
+
+easyblock = "MakeCp"
```
Comment (boegel) [inline easybuild/easyconfigs/s/samblaster/samblaster-0.1.21-goolf-1.5.14.eb]: above `name` please

File: easybuild/easyconfigs/s/samblaster/samblaster-0.1.21-goolf-1.5.14.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+homepage = "http://github.com/GregoryFaust/samblaster"
+description = """samblaster is a fast and flexible program for marking duplicates in read-id grouped paired-end SAM files. 
+It can also optionally output discordant read pairs and/or split read mappings to separate SAM files, and/or unmapped/clipped 
+reads to a separate FASTQ file. When marking duplicates, samblaster will require approximately 20MB of memory per 1M read pairs.
```
Comment (boegel) [inline easybuild/easyconfigs/s/samblaster/samblaster-0.1.21-goolf-1.5.14.eb]: continuation lines should start with a space

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/29b64216c1c9c82bc7b6 for a full test report.

**Comment by boegel** [comment]
@jordiblasco: looks good, just minor style remarks to tackle

File: easybuild/easyconfigs/s/samblaster/samblaster-0.1.21-goolf-1.5.14.eb (hunk 2 above)
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/samblaster/samblaster-0.1.21-goolf-1.5.14.eb]: please readjust description for readability, this is largely ok

File: easybuild/easyconfigs/s/samblaster/samblaster-0.1.21-goolf-1.5.14.eb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+toolchainopts = {}
+
+sources = ['v.%(version)s.tar.gz']
+source_urls = ['http://github.com/GregoryFaust/samblaster/archive']
+
+files_to_copy = [(['samblaster'], 'bin'), 'LICENSE.txt', 'README.md']
+
+sanity_check_paths = {
+    'files': ['bin/samblaster'],
+    'dirs': ["."]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/samblaster/samblaster-0.1.21-goolf-1.5.14.eb]: drop dot, we all know where that comes from :)

**Comment by boegel** [comment]
Jenkins: (re)test this please

**Comment by boegel** [comment]
I'd like to see the style remarks taken care of first.

**Comment by migueldiascosta** [comment]
@jordiblasco, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**